app/src/main/java/dev/arkbuilders/rate/presentation/quick/QuickViewModel.kt

```diff
@@ -56,6 +56,7 @@ class QuickViewModel(
     private val quickRepo: QuickRepo,
     private val prefs: Prefs,
     private val convertUseCase: ConvertWithRateUseCase,
+    private val calcFrequentCurrUseCase: CalcFrequentCurrUseCase,
```

Comment:
It looks like `assetsRepo`, `prefs`, and `convertUseCase` are not used in this ViewModel. Can we remove them?

Reply:
fixed